ember/app/ui/cases/index/controller.js

```diff
@@ -74,9 +139,20 @@ export default class CasesIndexController extends Controller {
   }
 
   @action
-  updateFilter(type, event) {
-    this[type] = event.target.value;
+  updateFilter(type, eventOrValue) {
+    if (eventOrValue.target) {
+      this[type] = eventOrValue.target.value;
+    } else if (type === "selectedIdentity") {
+      this[type] = eventOrValue.filterBy("idpId").mapBy("idpId");
+    }
+
+    this.fetchCases.perform();
+  }
+
+  @action
+  async updateIdentitySearch(value) {
+    this.identitySearch = value;
 
-    debounce({}, this.fetchCases.perform, 300);
+    await this.fetchIdentities.perform();
```

Comment:
Was the debounce not a good idea?

Reply:
debounce didnt seem to work/ too short, so switched it to timeout